src/rgw/rgw_rados.cc

```diff
@@ -6468,6 +6468,8 @@ int RGWRados::Object::Write::_do_write_meta(uint64_t size, uint64_t accounted_si
 
   bool versioned_op = (target->versioning_enabled() || is_olh || versioned_target);
 
+  struct rgw_bucket_dir_entry_meta dir_meta;
```

Comment:
@dongbula (NIT) Kindly consider to keep PR title < 50 charactershttps://github.com/ceph/ceph/blob/master/SubmittingPatches.rst#title-of-pull-requests-and-title-of-commits